script/add-payment-partials-to-marriage-abroad-services-data.rb

```diff
@@ -1,14 +1,26 @@
 countries = %w(
-  bulgaria
-  croatia
-  cyprus
-  guatemala
-  moldova
-  panama
-  venezuela
+  austria
+  brazil
+  czech-republic
+  denmark
+  ecuador
+  finland
+  germany
+  iceland
+  luxembourg
+  slovenia
+  sweden
 )
 
+# Exclude countries ignored in the outcome template
+countries = countries - %w(czech-republic cote-d-ivoire)
```

Comment:
Minor: I think it would be useful to explain that this was added because of this conditional statement in the outcome template: `elsif %w(czech-republic cote-d-ivoire).exclude?(calculator.ceremony_country)`. Either in the comment or the commit note.